packages/contracts/src/utils/JwtRegistry.sol

```diff
@@ -101,12 +92,23 @@ contract JwtRegistry is IDKIMRegistry, Ownable {
         dkimRegistry.revokeDKIMPublicKeyHash(publicKeyHash);
     }
 
+    function isAzpWhitelisted(
+        string memory azp
+    ) public view returns (bool) {
+        return whitelistedClients[azp];
+    }
+
+    function whitelistAzp(
+        string memory azp
+    ) public onlyOwner(){
```

Comment:
@wshino should this be `onlyOwner` and not `onlyOwner()`?

Reply:
@shreyas-londhe Sorry it's typo